test/e2e/new-dns-record.spec.ts

```diff
@@ -16,27 +20,93 @@ test.describe('authenticated as user', () => {
     await page.goto('/dns-records/new');
   });
 
-  test('redirects to edit DNS record page when required fields are filled', async ({ page }) => {
-    await page.getByLabel('Name*').fill('test');
-    await page.getByRole('combobox', { name: 'Type' }).selectOption('A');
-    await page.getByLabel('Value*').fill('test');
+  /**
+   * Clean up after all tests are run
+   * For some weird reason using afterEach to delete the one record doesn't work
+   * The test is run before the records are deleted
+   * Thus, we are using different values for the fields for the tests
+   */
+  test.afterAll(async () => {
+    await prisma.dnsRecord.deleteMany();
+  });
```

Comment:
This is caused locally due to a race condition as we run tests in parallel using multiple workers. A fix is to fix the number of workers locally to 1 to simulate what happens in CI.